cmd/fuzzcrypto/std/rsa_test.go

```diff
@@ -171,6 +170,26 @@ func FuzzRSASignPKCS1v15(f *testing.F) {
 	})
 }
 
+func validRSAError(err error) bool {
```

Comment:
Maybe call this `expectedRSAError`? To me an error can be "valid" (e.g. self-consistent), and still show up in unexpected circumstances.

It might be better to be more specific though... even in this file, this func is being used for some `err`s and not others, so there are still *some* places where it's expected vs. unexpected. Maybe:
`if errorCorrectlyRejectedRSAInput(err) { return }`